### DISCUSSION TITLE
What type of rails architecture does chatwoot have?

### DISCUSSION THREAD

**Discussion by contribute-rails-shubham-jain** [chatwoot/chatwoot#2642] [Q&A] [answered]
I mean to say that. Is it api only rails app?

Thank you

**Comment by contribute-rails-shubham-jain** [comment]
update, am I right ?

**Comment by sojan-official** [comment, marked as answer]
Chatwoot isn't exactly an API-only rails app. There are areas where we render the whole page using rails views 
example: Super Admin panel
